src/lib/libast/port/mnt.c

```diff
@@ -134,8 +134,8 @@ set(Header_t* hp, const char* fs, const char* dir, const char* type, const char*
  * what a crappy interface
  * data returned in static buffer -- ok
  * big chunk of allocated memory that cannot be freed -- come on
- * *and* NetBSD changed the interface somewhere along the line
- * private interface? my bad -- public interface? par for the BSD course
+ * *and* NetBSD changed the interface somewhere along the line to a
+ * private interface? par for the BSD course
```

Comment:
This edit makes the comment say something completely different; it is criticising NetBSD for changing a public interface. Either leave it as is, or delete lines 134-140 entirely.